docs/rst/getting_started/tutorial.rst

```diff
@@ -148,7 +157,7 @@ First of all, the number of entities discovered has increased.
 Now, you have a *DomainParticipant* called :code:`Participant_sub`, that holds a *DataReader* called
 :code:`DataReader_HelloWorldTopic_0.0.1.4`.
 This *DataReader* has a locator, which will be the *Shared Memory Transport* locator.
-That is because the Monitor and the *DomainParticipant* are running in the same host, and so they use *interprocess*
+That is because the Monitor and the *DomainParticipant* are running in the same host, and so they use *shared memory*
 communication.
```

Comment:
```suggestion
That is because the Monitor and the *DomainParticipant* are running in the same host, and so they communicate using the Shared Memory Transport (SHM) protocol.
```

Reply:
:+1: